apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -14,24 +14,26 @@ const initialState: AITutorState = {
   isWaitingForAIResponse: false,
 };
 
+interface question {
```

Comment:
nit:I think `interface`s are meant to be capitalized?

Reply:
I also wonder if a broader name for this is better, like `ChatContext`. (I think of "question" and "userMessage" -- a term used in Aichat for the next message the user sends to the endpoint -- as being equivalent.)